SPECS-EXTENDED/pcp/pcp.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "pcp-5.1.1.src.tar.gz": "f80246a360878339bf61b889d7c4cf8531547ce9f76b359b496860db89aad94a"
+  "pcp-5.1.1.src.tar.gz": "70b2cec444fbd6f6db5f544a84f93f436417a03cc3d1963c39b0525ac9b45609"
  }
```

Comment:
This might be a bit tricky to update in the blob store, since both files have the same name. One solution: update to slightly higher version?:)